core-bundle/src/Resources/contao/controllers/BackendPassword.php

```diff
@@ -81,8 +82,12 @@ public function run()
 			// Save the data
 			else
 			{
+				/** @var EncoderFactoryInterface $encoderFactory */
+				$encoderFactory = System::getContainer()->get('security.encoder_factory');
+				$encoder = $encoderFactory->getEncoder(BackendUser::class);
+
 				// Make sure the password has been changed
-				if (password_verify($pw, $this->User->password))
+				if ($encoder->isPasswordValid($this->User->password, $pw, null))
```

Comment:
Why did you change the argument order?

Reply:
Because that's what the interface expects.